lib/node_modules/@stdlib/stats/incr/wstdev/README.md

```diff
@@ -0,0 +1,173 @@
+
+<section class="intro">
+
+The [weighted sample standard deviation][weighted-sample-stdev] is defined as
+
+<!-- <equation class="equation" label="eq:weighted_sample_standard_deviation" align="center" raw="s = \sqrt{\frac{\displaystyle\sum_{i=0}^{n-1} w_{i} \left( x_{i} - \bar{x} \right)^2}{\displaystyle\sum_{i=0}^{n-1} w_{i}}}" alt="Equation for the weighted sample standard deviation."> -->
+
+```math
+s = \sqrt{\frac{\displaystyle\sum_{i=0}^{n-1} w_{i} \left( x_{i} - \bar{x} \right)^2}{\displaystyle\sum_{i=0}^{n-1} w_{i}}}
+```
+
+<!-- <div class="equation" align="center" data-raw-text="s = \sqrt{\frac{\displaystyle\sum_{i=0}^{n-1} w_{i} \left( x_{i} - \bar{x} \right)^2}{\displaystyle\sum_{i=0}^{n-1} w_{i}}}" data-equation="eq:weighted_sample_standard_deviation">
+    <img src="https://cdn.jsdelivr.net/gh/stdlib-js/stdlib@adbea9806383f70c982e3191475c874efba1296b/lib/node_modules/@stdlib/stats/incr/wstdev/docs/img/equation_weighted_sample_standard_deviation.svg" alt="Equation for the weighted sample standard deviation.">
+    <br>
+</div> -->
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var incrwstdev = require( '@stdlib/stats/incr/wstdev' );
+```
+
+#### incrwstdev()
+
+Returns an accumulator `function` which incrementally computes a [weighted sample standard deviation][weighted-sample-stdev].
+
+```javascript
+var accumulator = incrwstdev();
+```
+
+#### accumulator( \[x, w] )
+
+If provided an input value `x` and a weight `w`, the accumulator function returns an updated weighted sample standard deviation. If not provided any input values, the accumulator function returns the current weighted sample standard deviation.
+
+```javascript
+var accumulator = incrwstdev();
+
+var s = accumulator( 2.0, 1.0 );
+// returns 0.0
+
+s = accumulator( 3.0, 2.0 );
+// returns ~0.471
+
+s = accumulator( 2.0, 0.1 );
+// returns ~0.478
+
+s = accumulator();
+// returns ~0.478
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   Input values are **not** type checked. If provided `NaN` or a value which, when used in computations, results in `NaN`, the accumulated value is `NaN` for **all** future invocations. If non-numeric inputs are possible, you are advised to type check and handle accordingly **before** passing the value to the accumulator function.
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var uniform = require( '@stdlib/random/base/uniform' );
+var incrwstdev = require( '@stdlib/stats/incr/wstdev' );
+
+var accumulator;
+var v;
+var w;
+var i;
+
+// Initialize an accumulator:
+accumulator = incrwstdev();
+
+// For each simulated datum, update the weighted sample standard deviation...
+for ( i = 0; i < 100; i++ ) {
+    v = uniform( 0.0, 100.0 );
+    w = uniform( 0.0, 100.0 );
+    accumulator( v, w );
+}
+console.log( accumulator() );
```

Comment:
```suggestion
// Initialize an accumulator:
var accumulator = incrwstdev();

// For each simulated datum, update the weighted sample standard deviation...
var i;
for ( i = 0; i < 100; i++ ) {
    accumulator( uniform( 0.0, 100.0 ), uniform( 0.0, 100.0 ) );
}
console.log( accumulator() );
```

To avoid additional variables. Bigger picture, we want to make ES5 examples as easy to translate to ES6+ as possible in an automated fashion.